gui/default/syncthing/core/logViewerModalView.html

```diff
@@ -26,6 +27,18 @@
         </table>
       </div>
 
+      <div id="log-viewer-paths" class="tab-pane">
+        <label translate>Internally used paths:</label>
+        <table class="table table-striped">
+          <tbody>
+            <tr ng-repeat="(name, path) in logging.paths">
+              <td class="text-nowrap">{{ name | uncamel }}</td>
+              <td><code>{{ path }}</code></td>
```

Comment:
```suggestion
              <td class="file-path"><code>{{ path }}</code></td>
```

The GUI is broken on small screens due to wrapping not being allowed.

Before:

![image](https://user-images.githubusercontent.com/5626656/166834368-b83cf46e-6d64-4cd1-801b-8b8d4460b948.png)

After:

![image](https://user-images.githubusercontent.com/5626656/166834229-81ddb446-1726-4636-b549-9ef917427d09.png)